add secret_text, plain_text binding kinds (#695)
cloudflare/workers-docs: src/content/tooling/api/scripts.md

     -F "wasm=@module.wasm;type=application/wasm" # link your wasm module in place of module.wasm
```

#### Add a Secret Text Binding

> *Note: Secret text bindings are actually persisted between deploys, so if you are not changing the content of your secret, this is unnecessary, but also un-harmful.*

Reviewer: ```suggestion
*Note: Secrets are persisted between deploys of a Worker. You only need to include secrets in API calls when you are adding or changing the secret's content.*
```

Reviewer: Wording was confusing

> If your Worker uses encrypted environment variables, you will want to add a `secret_text` binding object for each one to the `"bindings"` array in `metadata.json`:

Reviewer: ```suggestion
If your Worker script uses [secrets](reference/apis/environment-variables#secrets), add a corresponding `secret_text` binding to the `"bindings"` array in `metadata.json`:
```

```json
{
  "body_part": "script",
  "bindings": [
    {
      "type": "secret_text",
      "name": "MY_SECRET",
      "text": "secret things are secret"
    }
  ]
}
```

> The `text` value should correspond to the text you want to store. The `name` value should correspond to the global variable you will use to access your secret from your Worker code.

Reviewer: ```suggestion
* `text` : the text you want to store
*  `name`:  the global variable to access your secret from your Worker code
```
